transforms-caching-experiment/README.md

```diff
@@ -0,0 +1,26 @@
+# Remote cache experiments for transforms
```

Comment:
The title could be a little more descriptive. As it is now, it's not clear what the experiment is for and why I would want to run this. I suggest something like `Artifact transform remote build cache performance experiment`.